### PR TITLE
CLI binaries doc

### PR DESCRIPTION
**Pull request by Q69K**

### CODE REVIEW

File: setup/cli.md [hunk 1]
```diff
@@ -6,37 +6,94 @@ description: Managing StackState using the CLI.
...
+### Windows
 
-The CLI can be downloaded from [https://download.stackstate.com](https://download.stackstate.com) using your license key.
+1. Download the executable `sts-cli-VERSION-windows.exe` from [https://download.stackstate.com](https://download.stackstate.com).
+2. Rename the downloaded file to be `sts.exe`.
+3. (optional) Place the file under your PATH to use StackState CLI commands from [anywhere on the command line](https://stackoverflow.com/questions/4822400/register-an-exe-so-you-can-run-it-from-any-command-line-in-windows).
+4. Follow the steps below to [launch the configuration wizard](#wizard-configuration-linux-and-windows-install) or [manually configure](#manual-configuration) the StackState CLI.
 
-The downloaded zip contains the following:
+### Docker (Linux, Windows, Mac)
```
Comment (mpvvliet) [inline setup/cli.md:29]: I am unclear about this section. Given the title Docker, I would expect it to explain how to run the CLI using a docker image? so possibly a Docker command line invocation.

Comment (mpvvliet) [reply setup/cli.md:29]: Is it missing a section on how to extract the ZIP file and then invoke a docker command?

Comment (Q69K) [reply setup/cli.md:29]: The `bin/sts` script is running Docker.

Comment (mpvvliet) [reply setup/cli.md:29]: Ah i get it now. By using the ZIP distribution, you will be using the shell script that invokes docker. That may be good to document. Also, it requires docker to be on the machine you run the script from.

**Review by mpvvliet — APPROVED** [review]

**Pull request merged**